Dockerfile_k8s_gpu

```diff
@@ -1,46 +1,51 @@
-# TODO(romilb) - The base image used here (ray) is very large (11.4GB).
-#  as a result, this built image is about 13.5GB. We need to pick a lighter base
-#  image.
-FROM rayproject/ray:2.9.3-py310-gpu
+FROM nvidia/cuda:12.1.1-runtime-ubuntu20.04
```

Comment:
Can we add a note here that we are using `runtime` instead `devel` because the latter is larger?